src/components/Pages/Symptoms/SymptomsList.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+// * Components
+import SymptomCard from "../../Organisms/SymptomCard";
+
+function SymptompsList() {
+  return (
+    <div className="symptomsList">
+      <h2>Síntomas del Covid-19</h2>
+      <div className="symptomsList__cards">
+        <SymptomCard
+          title={"Title"}
+          description={"Lorem algo algo asjdahsdiawd"}
+          image={"https://os.alipayobjects.com/rmsportal/QBnOOoLaAfKPirc.png"}
+        />
```

Comment:
In the List of Cards for "Symptoms" we just need to import the card component, ez game 👻